src/index.js

```diff
@@ -1,15 +1,16 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import Header from './components/Header'
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import Header from "./components/Header";
+import SideNav from "./components/CustomSidebar";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <Header />
+    <React.StrictMode>
+      <Header/>
+      <SideNav />
```

Comment:
Instead of adding   `<Header/>` and  `<SideNav />` and all other components in `index.js`, do it in `App.js` file